package.json

```diff
@@ -46,5 +46,9 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "engines": {
```

Comment:
Place at the top just above dependencies

Reply:
Done